Jenkinsfile

```diff
@@ -85,6 +85,7 @@ pipeline {
   }
   environment {
     CARGO_INCREMENTAL = "0"
+    PRODUCT_PREFIX = 'mayastor'
```

Comment:
Maybe then here should also be IMAGE_PREFIX ?

Reply:
Eventually this value would we used with the kubectl-mayastor binary to decide on the suffix name.

Reply:
Does IMAGE_PREFIX still fit?

Reply:
hmm good point, maybe leave this one as is then